Submitted By:            Miklos Karacsony <mkaracsony81 at gmail dot com>
Date:                    2014-04-22
Initial Package Version: 1.4.7
Upstream Status:         Not submitted.
Origin:                  Self
Description:             Match declaration types with the ones included in fontsproto-2.1.3

diff -U 3 -H -d -r -N -- libXfont-1.4.7/src/util/patcache.c libXfont-1.4.7.mod/src/util/patcache.c
--- libXfont-1.4.7/src/util/patcache.c	2014-01-07 16:25:08.000000000 +0000
+++ libXfont-1.4.7.mod/src/util/patcache.c	2014-04-22 05:00:45.500449766 +0100
@@ -128,7 +128,7 @@
 /* add entry */
 void
 CacheFontPattern (FontPatternCachePtr cache,
-		  char *pattern,
+		  const char *pattern,
 		  int patlen,
 		  FontPtr pFont)
 {
@@ -174,7 +174,7 @@
 /* find matching entry */
 FontPtr
 FindCachedFontPattern (FontPatternCachePtr cache,
-		       char *pattern,
+		       const char *pattern,
 		       int patlen)
 {
     int				hash;
